common/odp/odp.go

```diff
@@ -89,3 +89,13 @@ func AddDatapathInterface(dpname string, ifname string) error {
 	_, err = dp.CreateVport(odp.NewNetdevVportSpec(ifname))
 	return err
 }
+
+func DatapathSupported() bool {
```

Comment:
This PR removed the previous way it was checked, which was creating and then removing a datapath.  I don't actually recall why that was removed, but it is currently related.
Maybe I should just put exactly the same code back?
Why was the shell-script code rewritten to Go code in 24d9a00f471f6c879c7197306748cab1bf310ab7 ?